lib/auth_web/controllers/auth_controller.ex

```diff
@@ -95,6 +111,25 @@ defmodule AuthWeb.AuthController do
     {:ok, profile} = ElixirAuthGithub.github_auth(code)
     # save profile to people:
     person = Person.create_github_person(profile)
+
+    # login log
+    user_agent =
+      conn
+      |> get_user_agent_string()
+      |> Auth.UserAgent.get_or_insert_user_agent()
+
+    ip_address = get_ip_address(conn)
+
+    Auth.LoginLog.create_login_log(
+      %{
+        email: person.email,
+        ip_address: ip_address,
+        auth_provider: "github"
```

Comment:
technically we don't _care_ about people _attempting_ to login with their GitHub account.
they can _fail_ at attempting to login with GitHub as many times as they like.
It's the `email+password` login attempts we care about because these are people who have registered with our Auth system. We want to prevent automated login attempts.